### DISCUSSION TITLE
Reorganize the javacc repo

### DISCUSSION THREAD

**Discussion by new-javacc** [javacc/javacc#242] [Polls]

**Comment by vlsi** [comment]
How "separate modules" differs from "separate folders"?

Would you please correct the vote so there are just two answers: separate repositories vs single repository?

**Comment by vlsi** [comment]
For reference,  here are the reasons to proceed with single repository instead of multiple ones: https://github.com/javacc/javacc/discussions/238#discussioncomment-3109619, https://github.com/javacc/javacc/discussions/238#discussioncomment-3109644

**Comment by new-javacc** [comment]
Hey those are all your opinions - not reasons so if you have a problem - 
go ahead fork it and I won's ask to be a contributor lol

**Reply by martinswanson** [reply]
These kind of comments are really unnecessary. Please try to respect others. 
https://opensource.com/open-source-way

**Reply by new-javacc** [reply]
> These kind of comments are really unnecessary. Please try to respect others. https://opensource.com/open-source-way

When provoked that what happens. I don't care about OSS except for the fact that this is the only way to extract it from Sun. And if people have issues with it - they can fork and do/run it the way they want. 

I don't care a single about most of the issues brought out by @vlsi and disparaging the hard working Francis and not being helpful. So there I said it. For me this is such a useless wasted time.

Also I don't know why/how I merged that one PR that @vlsi  keeps harping about. There shouldn't be any loops. That 42 is a strange number. I'm glad that PR is not in 8.

This is such a waste precious little time I have to wotrk on this project. I meant every word of my comment above.

**Comment by martinswanson** [comment]
Sorry Sreeni, but I can’t be part of the community when you keep this attitude. It feels like bullying to me. 

This is not an OSS project. It is your private codebase that you don’t maintain properly, and block others from fixing. 

Best of luck to you and the rest of the community (sincerely).

Martin

**Comment by new-javacc** [comment]
Sorry but like I said I never blocked anyone from any of the work including core. But I don't give the same value to people who keep giving crap to me and Francis who tried to do something without being constructive and demanding random things that we don't have time for.On Jul 11, 2022 1:41 PM, Martin Swanson ***@***.***> wrote:

Sorry Sreeni, but I can’t be part of the community when you keep this attitude. It feels like bullying to me. 



This is not an OSS project. It is your private codebase that you don’t maintain properly, and block others from fixing. 



Best of luck to you and the rest of the community (sincerely).



Martin

**Reply by vlsi** [reply]
> Sorry but like I said I never blocked anyone from any of the work including core

This is false.
Here's a comment https://github.com/javacc/javacc-8/discussions/2#discussioncomment-3216593 saying that everyone except Sreeni is forbidden from committing code to core. At the same time, the only "allowed committer" makes no progress on improving the core, and they allocate no time for reviewing PRs as well.

> [kaikalur](https://github.com/kaikalur): The core is a mess and **I don't have time to review PRs that touch core**. So **I want to control what goes in there** whereas the code generators are open and free for all.
> A compromise maybe to have and enforce **OWNER file for core which currently will be only me** until we refactor/clean it up for easier management.
